src/openms/source/MATH/MISC/MIDAsFFT.cpp

```diff
@@ -0,0 +1,71 @@
+#include <OpenMS/MATH/MISC/MIDAsFFT.h>
+
+
+#define INVERSE
+using namespace std;
+
+namespace OpenMS
+{
+
+  void fft(double *Data, int nn)
```

Comment:
does it provide any benefits over functions from kissfft that have been added since then?